### ISSUE TITLE
HTTP chunk encoding

### ISSUE THREAD

**Issue by GooG2e** [yrutschle/sslh#244]
Hello!
I have problem with sslh connection
Some http transfers return as chunked with errors.
I don't see something strange in logs 
What information to diagnose this?
Thank you!

**Comment by yrutschle** [comment]
Yes, what information do you use to say there is something wrong?

**Comment by GooG2e** [comment]
> 
> 
> Yes, what information do you use to say there is something wrong?

If I connect direct to web server I receive full content, but when I use sslh I receive part of data with this message.
@yrutschle

**Comment by GooG2e** [comment]
@yrutschle may this be misconfiguration of nginx because I use nginx as proxy_pass to other web server

**Comment by GooG2e** [comment]
Checked nginx logs - in logs I see that nginx sends bigger information then client receives.
Nginx write that sent 22Mb and client receive only 10Mb
